Model/Indexer/Pages.php

```diff
@@ -55,6 +60,10 @@ public function executeRow($slug)
 
     private function addPagesForSync($slugs)
     {
+
+        if ($this->storeAutocompleteConfig->hasToSyncPages() == 0) {
```

Comment:
No need to compare. Function either returns true or false. So comparison can be ignored.